### PR TITLE
feat: Update item urns on URN change

### PR DESCRIPTION
**Pull request by LautaroPetaccio**
This PR updates the URN of the items of a collection that was saved.

### CODE REVIEW

File: src/modules/item/reducer.ts [hunk 1]
```diff
@@ -305,6 +307,39 @@ export function itemReducer(state: ItemState = INITIAL_STATE, action: ItemReduce
             }
           }
         }
+        case SAVE_COLLECTION_SUCCESS: {
+          const collectionURN = decodeURN(transaction.payload.collection.urn)
+
+          collectionURN.suffix
```
Comment (nicosantangelo) [inline src/modules/item/reducer.ts:313]: whops!

File: src/lib/urn.ts [hunk 2]
```diff
@@ -34,7 +34,7 @@ const protocolMatcher = '(?<protocol>mainnet|ropsten|matic|mumbai|off-chain)'
 const typeMatcher = '(?<type>base-avatars|collections-v2|collections-thirdparty)'
 
 const baseAvatarsSuffixMatcher = '((?<=base-avatars:)BaseMale|BaseFemale)'
-const collectionsSuffixMatcher = '((?<=collections-v2:)0x[a-fA-F0-9]{40})'
+const collectionsSuffixMatcher = '((?<=collections-v2:)(?<collectionAddress>0x[a-fA-F0-9]{40}))?(:(?<tokenId>[^:|\\s]+))'
```
Comment (nicosantangelo) [inline src/lib/urn.ts:37]: I think this is missing an `=` after each `<`
```suggestion
const collectionsSuffixMatcher = '((?<=collections-v2:)(?<=collectionAddress>0x[a-fA-F0-9]{40}))?(:(?<=tokenId>[^:|\\s]+))'
```

And that might be breaking the tests

**Review by nicosantangelo — CHANGES_REQUESTED** [review]
Broken tests!

File: src/modules/item/reducer.ts (hunk 1 above)
Comment (LautaroPetaccio) [reply src/modules/item/reducer.ts]: Removed it!

File: src/lib/urn.ts (hunk 2 above)
Comment (LautaroPetaccio) [reply src/lib/urn.ts]: The equal sign matches there the exact string (at least as far as I understood), but that's not needed for the collectionAddress nor for the tokenId.

Comment (nicosantangelo) [reply src/lib/urn.ts]: Correct! my bad

**Pull request merged**